packages/orchestrator-ui-components/src/components/WFOStartTaskButtonComboBox/WFOStartTaskButtonComboBox.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+
+import { useTranslations } from 'next-intl';
+import { useQueryWithGraphql } from '../../hooks';
+import { GET_WORKFLOWS_FOR_DROPDOWN_LIST_GRAPHQL_QUERY } from '../../graphqlQueries/workflows/workflowsQueryForDropdownList';
+import {
+    WorkflowComboBoxOption,
+    WFOButtonComboBox,
+} from '../WFOButtonComboBox/WFOButtonComboBox';
+import { PATH_START_WORKFLOW } from '../WFOPageTemplate';
+
+export const WFOStartTaskButtonComboBox = () => {
+    const router = useRouter();
+    const t = useTranslations('common');
+
+    const { data } = useQueryWithGraphql(
+        GET_WORKFLOWS_FOR_DROPDOWN_LIST_GRAPHQL_QUERY,
+        {
+            // Avoiding pagination by fetching an unrealistic amount of items
+            first: 1000,
+            after: 0,
+            filterBy: [{ field: 'target', value: 'SYSTEM' }],
+        },
+        'taskWorkflows',
+    );
+
+    const taskList: WorkflowComboBoxOption[] =
+        data?.workflows.page
+            .map((workflow) => {
+                return {
+                    label: workflow.description,
+                    data: { workflowName: workflow.name },
+                } as WorkflowComboBoxOption;
```

Comment:
The cast is not needed anymore and you can remove the braces from this oneliner-return statement